src/app/units/states/tasks/inbox/directives/staff-task-list/staff-task-list.component.html

```diff
@@ -171,7 +171,7 @@
   <!--/search-options-->
   @if (filteredTasks) {
   <cdk-virtual-scroll-viewport class="tasks-viewport scrollable" itemSize="60">
-    <mat-list>
+    <mat-list style="margin: 0; padding: 0">
```

Comment:
Please use the tailwind classes, `m-0 p-0`

Reply:
Done